plugins/modules/ndo_virtual_port_channel_interface.py

```diff
@@ -0,0 +1,383 @@
+        - The name of the Virtual Port Channel Interface.
+        type: str
+        aliases: [ name, virtual_port_channel, vpc ]
+    virtual_port_channel_interface_uuid:
+        description:
+        - The UUID of the Virtual Port Channel Interface.
+        - This parameter is required when parameter O(virtual_port_channel_interface) is updated.
+        type: str
+        aliases: [ uuid, virtual_port_channel_uuid, vpc_uuid ]
+    description:
+        description:
+        - The description of the Virtual Port Channel Interface.
+        type: str
+    node_1:
+        description:
+        - The first node ID.
+        type: str
+    node_2:
+        description:
+        - The second node ID.
+        type: str
+    interfaces_node_1:
+        description:
+        - The list of used Interface IDs for the first node.
+        - Ranges of Interface IDs can be used.
+        type: list
+        elements: str
+        aliases: [ interfaces_1, members_1 ]
+    interfaces_node_2:
+        description:
+        - The list of used Interface IDs for the second node.
+        - Ranges of Interface IDs can be used.
+        type: list
+        elements: str
+        aliases: [ interfaces_2, members_2 ]
+    interface_policy_group_uuid:
+        description:
+        - The UUID of the Port Channel Interface Setting Policy.
+        type: str
+        aliases: [ policy_uuid, interface_policy_uuid ]
+    interface_policy_group:
+        description:
+        - The name of the Port Channel Interface Policy Group.
+        - This parameter can be used instead of O(interface_policy_group_uuid).
+        type: dict
+        suboptions:
+            name:
+                description:
+                - The name of the Port Channel Interface Setting Policy.
+                type: str
+            template:
+                description:
+                - The name of the template in which is referred the Port Channel Interface Policy Group.
+                type: str
+        aliases: [ policy, interface_policy ]
+    interface_descriptions:
+        description:
+        - The list of descriptions for each interface.
+        type: list
+        elements: dict
+        suboptions:
+            node:
+                description:
+                - The node ID.
+                type: str
+            interface_id:
+                description:
+                - The interface ID.
+                type: str
+            description:
+                description:
+                - The description of the interface.
+                type: str
+    state:
+        description:
+        - Use C(absent) for removing.
+        - Use C(query) for listing an object or multiple objects.
+        - Use C(present) for creating or updating.
+        type: str
+        choices: [ absent, query, present ]
+        default: query
+extends_documentation_fragment: cisco.mso.modules
+"""
+
+EXAMPLES = r"""
+- name: Create a new Virtual Port Channel Interface
```

Comment:
examples for uuid update/query/delete are missing

Reply:
will add them